### ISSUE TITLE
Issue with Downloading Jira Data from Non Apache Project

### ISSUE THREAD

**Issue by lh-zhan** [sailuh/kaiaulu#256]
When attempting to download JIRA data from non apache project such as Hibernate: https://hibernate.atlassian.net/jira/software/c/projects/HHH/issues/

Domain for regular Apache project doesn't work: e.g. https://hibernate.atlassian.net/jira/, causing `Error: API call returns non JSON data` 

Had to debug and found if we use all issues link by removing `/jira/`, data can be downloaded for this specific project.

Might be beneficial to create wrappers that can attempt various possible endpoints automatically.

**Comment by carlosparadis** [comment]
@lh-zhan 

I don't believe Kaiaulu assumes the domain to contain `/jira`. This is a parameter you have to specify on your project configuration file: 

e.g. https://github.com/sailuh/kaiaulu/blob/ffd5f300e0c66c70ccc2beefcb7ab7da9ee9408c/conf/geronimo.yml#L58

Am I missing something here?

**Comment by lh-zhan** [comment]
Yeah, I guess I just wanted to point out that sometimes it's without the JIRA keyword even thought it's an JIRA issue board and in the JIRA section of the config file. Might be worth making a documentation to save the debugging process for the users : )

But other than that, you can close this issue.

**Comment by carlosparadis** [comment]
No worries. This should be addressed when I update the kaiaulu JIRA config, which does not require `/jira`. I will mark this as invalid so I don't think it is a bug. 

I guess the more general question is how to document the more bizarre situations like Spring migration. If not in a way that can automate it, to at least warn on the project configuration file, since it is not viable to have a notebook for every situation.

**Comment by lh-zhan** [comment]
Yeah I agree, probably just general procedures of how to make new functions in `metrics.R` files to accommodate different fields in new issue data.
